.github/workflows/build_navitia_packages_for_dev_multi_distribution.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     - name: remove useless temporary files
       run: rm -rf ../navitia-*
     - name: slack notification (the job has failed)
-      if: failure()
+      if: ${{ failure() && github.event_name == 'push' }}
```

Comment:
I use this kind of syntax : https://github.com/CanalTP/navitia/blob/dev/.github/workflows/build_navitia_packages_for_release.yml#L57

Reply:
Ok, that's more readable :+1: